MyQuizAdmin/MyQuizAdmin/Model/GroupModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace MyQuizAdmin.Model
+{
+    public class Group
+    {
+        public int id;
+        public string name;
```

Comment:
Please be aware of the [group model](https://github.com/Kulu-M/MyQuizBackend/blob/master/MyQuizBackend/src/MYVote/Models/Group.cs) in the backend